Kickstarter-iOS/Features/RemoteConfigFeatureFlagTools/Controller/RemoteConfigFeatureFlagToolsViewController.swift

```diff
@@ -53,15 +53,6 @@ final class RemoteConfigFeatureFlagToolsViewController: UITableViewController {
 
   @objc private func switchToggled(_ switchControl: UISwitch) {
     self.viewModel.inputs.setFeatureAtIndexEnabled(index: switchControl.tag, isEnabled: switchControl.isOn)
-
-    let (feature, _) = self.features[switchControl.tag]
-    if feature == RemoteConfigFeature.creatorDashboardEnabled {
-      NotificationCenter.default.post(
-        name: Notification.Name.ksr_userLocalePreferencesChanged,
-        object: nil,
-        userInfo: nil
-      )
-    }
```

Comment:
This still might be worth having for testing. With your solution here, nothing will happen when toggling the feature from the local ff menu